src/ZoneServer/Network/Send.Normal.cs

```diff
@@ -541,6 +541,23 @@ public static void SetHitDelay(Character character, int skillId, float value)
 				character.Connection.Send(packet);
 			}
 
+			/// <summary>
+			/// Reset skill ?
+			/// SKL_CANCEL_CANCEL
+			/// </summary>
+			/// <param name="actor"></param>
+			/// <param name="skillId"></param>
+			public static void SkillCancelCancel(IActor actor, SkillId skillId)
```

Comment:
I changed the comment a bit, since we now have a bit better of an idea of what this packet does.  I don't know if you prefer to rename it, we can see from skill bytool that this actually is its name, so maybe it should stay that way.

Reply:
Fair enough 👍